editoast/Cargo.toml

```diff
@@ -48,3 +48,4 @@ editoast_derive = { path = "./editoast_derive" }
 mvt = "0.8.1"
 pointy = "0.4.0"
 futures = "0.3.26"
+ntest = "0.9.0"
```

Comment:
Shouldn't it be under [`dev-dependencies`](https://doc.rust-lang.org/rust-by-example/testing/dev_dependencies.html) ?